core/templates/dev/head/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.html

```diff
@@ -140,15 +140,23 @@
           </a>
           <div ng-if="activeHintIndex === $index">
             <div class="oppia-editor-card-section protractor-test-hint-<[$index]>-text">
-              <angular-html-bind html-data="getRequiredHtml(hint.hintContent)" class="oppia-rte-editor">
-              </angular-html-bind>
+              <span ng-show="!getRequiredHtml(hint.hintContent)" class="oppia-placeholder">
```

Comment:
We generally use `ng-if`, so can you change that?

Reply:
Yeah can do that. Is this for making the angularjs to angular 8 migration ready.

Reply:
Not really, with `ng-if`, the element is not rendered at all, but `ng-show`, only changes the CSS to `display: none;`.

Reply:
Ok

Reply:
done.